### ISSUE TITLE
More Output Formats

### ISSUE THREAD

**Issue by steffenf14** [editorconfig-checker/editorconfig-checker#245]
Hi, 
are there plans to add more output formats like junit?
My goal is to have a better integration in gitlab pipelines for example.
I have seen such implementations in deptrac or php codesniffer

**Comment by mstruebing** [comment]
I did not plan it yet.
If someone is willing to provide a PR I will sure have a look at it and merge it if it's good.

**Comment by jerone** [comment]
Issue #317 requests an output for GitHub Actions.

**Comment by jerone** [comment]
Besides the mentioned formats (junit & GitHub Actions), I would like to ask for API to define **custom formats** myself.

I'm working in a project with multiple linters and I'm writing custom formatters/outputs/reporters for them to return an constant output logging.
So far I was able to write reporters for [CSpell](https://cspell.org/types/cspell-types/interfaces/CSpellReporter.html) & [ESLint](https://eslint.org/docs/latest/extend/custom-formatters).

**Comment by KaffeMyers** [comment]
I think the format that is used by flake8 (and a lot of other linters) should be one of the standard options, as it is an established output standard used by several linters and is easy to parse:
``` shell
$ flake8
./job-generator.py:1:5: E116 unexpected indentation (comment)
```
